backend/Squirrel.Core/Squirrel.Core.Common/Enums/ErrorType.cs

```diff
@@ -5,5 +5,6 @@ public enum ErrorType
     InvalidEmail = 1,
     InvalidUsername,
     InvalidPassword,
```

Comment:
For security reasons we shouldn't say the email or password was not valid, probably we can remove `InvalidPassword`?